htcanalyze/main.py

```diff
@@ -21,15 +21,22 @@
 
 import configargparse
 
+# import own classes
 from htcanalyze.htcanalyze import HTCAnalyze, raise_value_error
 from htcanalyze.logvalidator import LogValidator
+# import for colorful output
 from rich import print as rprint, box
 from rich.progress import Table
 
-# typing identities
+
+# after analyzing, the information for each log file is saved
+# in it's on dictionary
 log_inf_list = List[dict]
+# dictionary with contents of all valid log files as a string
 list_of_logs = List[str]
+# save start of script
 date_time = datetime.datetime
+# save runtime of script
```

Comment:
Same as above. Commenting is good, but should explain /why/, not what. variable names explain the "what".

If `timedelta` is unclear, it should be better named.

Reply:
Same as above, it doesn't actually do this. Same comment applies to doing redeclarations in the imports.

`from datetime import datetime as date_time, timedelta` has the same effect. Be careful with `from datetime import datetime`, for obvious reasons.